doc/topics/mine/index.rst

```diff
@@ -42,13 +42,57 @@ the example below:
       test.ping: []
       network.ip_addrs:
         interface: eth0
-        cidr: '10.0.0.0/8'
+        cidr: 10.0.0.0/8
 
 In the example above :py:mod:`salt.modules.network.ip_addrs` has additional
 filters to help narrow down the results.  In the above example IP addresses
 are only returned if they are on a eth0 interface and in the 10.0.0.0/8 IP
 range.
 
+.. versionchanged:: Sodium
+
+The format to define mine_functions has been changed to allow the same format
+as used for module.run. The old format (above) will still be supported until
```

Comment:
The old format will still be supported. Please remove the until.... bit.

Reply:
Updated in bfee1c8d9bcab7685bbc95f5d05a493db52560e9